aws/resource_aws_appsync_schema.go

```diff
@@ -0,0 +1,87 @@
+package aws
+
+import (
+	"errors"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/appsync"
+	"github.com/hashicorp/terraform/helper/schema"
+	"time"
+)
+
+func resourceAwsAppsyncSchema() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsAppsyncSchemaPut,
+		Read:   resourceAwsAppsyncSchemaNil,
+		Update: resourceAwsAppsyncSchemaPut,
+		Delete: resourceAwsAppsyncSchemaNil,
```

Comment:
Not having a deletion method may be another sign that this may not make sense as its own resource, but instead an attribute of an existing "parent" resource (`aws_appsync_graphql_api`). 👍  Otherwise, we should use [`schema.Noop`](https://godoc.org/github.com/hashicorp/terraform/helper/schema#Noop) instead of our own function here and add a warning to the documentation that removing this resource does not actually delete the schema.